sql-aggregates/total-dvds.sql

```diff
@@ -0,0 +1,3 @@
+select count("inventory") as "totalDVDs"
```

Comment:
The argument to `count` is actually ignored, so it's traditional to always use `*`: `count(*)`.